datafusion-examples/examples/flight/flight_sql_server.rs

```diff
@@ -246,27 +276,6 @@ impl FlightSqlService for FlightSqlServiceImpl {
         Ok(Response::new(Box::pin(stream)))
     }
 
-    async fn get_flight_info_statement(
```

Comment:
removing this code means it falls back to the default implementation (which also returns Not implemented)

Seems like a nice change to me 👍